templates/github/.github/workflows/create-branch.yml.j2

```diff
@@ -13,6 +13,15 @@ on:
       name:
         description: "Branch name (e.g. 3.14)"
         required: true
+      update_ci_branches:
+        description: "Behavior for updating ci_update_branches"
+        required: true
+        type: choice
+        default: "none"
+        options:
+          - "append-version"
+          - "replace-previous-version"
+          - "none"
```

Comment:
Interesting idea, but i wonder if we should make this choice in the template-config. Giving the user choice usually leads to confusion.
I suspect we will hear: "Did you not select this option? We always do. Now I need to do it by hand."

Reply:
Like another variable to decide the behavior? I was also thinking of just having the default be different for pulpcore, have it be set to replace-previous-version instead of none since this is the behavior we've decided on for our release policy.

Reply:
What do you suggest then? Switch it to a template-config variable?